addons/mail/static/src/components/attachment/attachment.scss

```diff
@@ -43,6 +43,14 @@
     justify-content: center;
 }
 
+.o_Attachment_button {
+    background: none;
+    color: inherit;
+    border: none;
+    padding: 0;
+    cursor: pointer;
+}
```

Comment:
It's really annoying that having the `button` tag comes with so much crap that we have to clean...

Can we not change it, and use tabindex or whatnot to make the div focusable instead?

Reply:
I agree. Removed and changed div by links that don't require extra CSS.